common/test/common/ExperimentsTest.scala

```diff
@@ -13,10 +13,6 @@ class ExperimentsTest extends AnyFlatSpec with Matchers {
 
   conf.switches.Switches.ServerSideExperiments.switchOn()
 
-  "Experiments" should "not share participation group" in {
```

Comment:
I feel a bit uneasy about deleting this test. If it is a blocker could we comment it out just for the duration of this test instead as it is probably a good thing to warn about overlapping participations.

Reply:
This was a temporary commit so that a build would complete and I could test the AB test functionality on CODE.

I was going to ask what the team thought on the best way to progress. I assume adding more groups [here](https://github.com/guardian/fastly-edge-cache/blob/0c366d7f8ef16a5b664fe7205cdd4bae57e07f56/theguardiancom/src/main/resources/varnish21/ab-tests.vcl) and editing the configuration in Fastly would be too much faff. I'll comment it out for the duration of the AB test instead.